### PR TITLE
Added WSL instructions to readme

### LINKED ISSUE #27: STEMMUS_SCOPE executable file incompatible with Windows
After installing Matlab runtime, I get the following error trying to run the executable file in Windows:

`The specified executable is not a valid application for this OS platform`

We should figure out what is required to generate an executable file that is compatible with Windows, or how to properly run the executable file on Windows.

### PR DESCRIPTION
**Pull request by BSchilperoort**
Added concise instructions for Windows users on how to set up WSL to be able to run PyStemmusScope.

Closes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,34 @@ instructions for `Users` or `Developers` on how to run the model.
 
 As a user, you don't need to have a MATLAB license to run the STEMMUS-SCOPE model. The workflow is executed using python and MATLAB Runtime on a Unix-like system.
 
-### Installations 
+As the STEMMUS-Scope executable only supports Unix-like systems, Windows users cannot run STEMMUS-SCOPE natively.
```
Comment (SarahAlidoost) [inline README.md:29]:
```suggestion
As the STEMMUS-SCOPE executable only supports Unix-like systems, Windows users cannot run STEMMUS-SCOPE natively.
```

Comment (SarahAlidoost) [reply README.md:29]: Can we add the link to this [MATLAB documentation](https://nl.mathworks.com/help/compiler/standalone-applications.html) where the reason is explained? More specifically see the line `Standalone applications run on Windows®, Linux®, and macOS corresponding to the platform on which they are generated.`

File: README.md [hunk 2]
```diff
@@ -26,7 +26,34 @@ instructions for `Users` or `Developers` on how to run the model.
...
+However, users of Windows 10 and newer can use WSL (Windows Subsystem for Linux) to run the model.
```
Comment (SarahAlidoost) [inline README.md:30]:
```suggestion
However, users of Windows 10 and newer can use WSL ([Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)) to run the model.
```

File: README.md [hunk 3]
```diff
@@ -26,7 +26,34 @@ instructions for `Users` or `Developers` on how to run the model.
...
+
+  After installation, you can start up the WSL instance and update the default software:
+
+  ```sh
+  sudo apt update && sudo apt upgrade
+  ```
+
+  You can now set up a python environment using either python's `venv`, or use Conda/Mamba.
+
+  Note that the command to run python and pip can be `python3` and `pip3` by default.
```
Comment (SarahAlidoost) [inline README.md:49]:
```suggestion
  You can now set up a python environment using either python's `venv`, or use Conda/Mamba.
  Note that the command to run python and pip can be `python3` and `pip3` by default.
```

**Review by SarahAlidoost — APPROVED** [review]
@BSchilperoort thanks, looks good to me. Just some small suggestions, see my comments. Also, can we link this PR to issue #27? I think this PR can close the issue.

File: README.md (hunk 1 above)
Comment (BSchilperoort) [reply README.md:29]: It is more complex than just that. If we had a matlab license I could just compile the model on my machine whenever a new STEMMUS-SCOPE version is released. I think that keeping it simple and concise here might be better.

Comment (SarahAlidoost) [reply README.md]: Besides matlab license, creating one executable file per platform is due to the fact that MATLAB does not support cross-platform compilers. Adding the documentation link provides more information for the maintainers of the codes in the future. :) perhaps we can add the link to the readme of [STEMMUS_SCOPE/exe](https://github.com/EcoExtreML/STEMMUS_SCOPE/tree/main/exe) later, see issue https://github.com/EcoExtreML/STEMMUS_SCOPE/issues/28

**Pull request merged**